pkg/api/v20210131preview/openshiftcluster.go

```diff
@@ -149,15 +149,22 @@ const (
 	VMSizeStandardD16sV3 VMSize = "Standard_D16s_v3"
 	VMSizeStandardD32sV3 VMSize = "Standard_D32s_v3"
 
-	VMSizeStandardE4sV3  VMSize = "Standard_E4s_v3"
-	VMSizeStandardE8sV3  VMSize = "Standard_E8s_v3"
-	VMSizeStandardE16sV3 VMSize = "Standard_E16s_v3"
-	VMSizeStandardE32sV3 VMSize = "Standard_E32s_v3"
+	VMSizeStandardE4sV3   VMSize = "Standard_E4s_v3"
+	VMSizeStandardE8sV3   VMSize = "Standard_E8s_v3"
+	VMSizeStandardE16sV3  VMSize = "Standard_E16s_v3"
+	VMSizeStandardE32sV3  VMSize = "Standard_E32s_v3"
+	VMSizeStandardE64isV3 VMSize = "Standard_E64is_v3"
+	VMSizeStandardE64iV3  VMSize = "Standard_E64i_v3"
```

Comment:
New VM types necessary for [compute isolation](https://docs.microsoft.com/en-us/azure/security/fundamentals/isolation-choices)